### PR TITLE
feat: darkMode for user profile has been done

### LINKED ISSUE #109: [FEATURE 💡] Dark Mode for User Profile
### Description

+ Add dark mode to user profile
+ Match the appearance of the existing UI

### Screenshots

Screenshot_2023-10-06-20-57-46-818_com android chrome

### Additional information

- Always use styled components instead of plain CSS and MUI components.
## Happy contributing. 💐
Star my other Repositories [here](https://github.com/warmachine028)

### PR DESCRIPTION
**Pull request by waleedkhano**
# Issue #109 📐

- I worked on feature #109

## Guidelines 🔐

I accept the fact that I have followed the guidelines and have not copied the codes from around the internet

## Issue to be closed 🛅

- My pull request closes #109

## Screenshots 📷

Here are the video of changes that I have made 🔽

https://github.com/warmachine028/memories/assets/110036553/6bad3cf4-f549-4dcf-93db-7baabedbd308

### CODE REVIEW

**Comment by warmachine028** [comment]
Pretty quick, quite impressive work. I'll review it

**Review by warmachine028 — CHANGES_REQUESTED** [review]
Instead of doing it in inline styles, could you please work on the classes.component comming from styles.js? 
The inline styles surely get the job done but it clutters the code for developer's

**Comment by waleedkhano** [comment]
Sure

**Comment by waleedkhano** [comment]
@warmachine028 please review the code in which I replaced the inline styles with classes.

**Review by warmachine028 — CHANGES_REQUESTED** [review]
- image

These texts must not appear white in Light mode, the original color was alright.

- image
These card texts are not properly visible in dark mode, please fix them too

- image

the title of these posts in commnents would look better in white for only dark mode.

**Comment by waleedkhano** [comment]
Could you please specify the location of the comments and the card?

it looks ok here
card


 
comments

**Comment by warmachine028** [comment]
https://github.com/warmachine028/memories/tree/main/client/src/components/PostDetails/CommentSection

https://github.com/warmachine028/memories/tree/main/client/src/components/PostDetails/CommentSection

**Comment by waleedkhano** [comment]
card text has been changed to white



comment title text also changed

**Review by warmachine028 — CHANGES_REQUESTED** [review]
Build failed, please see the Logs

Screenshot_2023-10-08-14-59-14-526_com android chrome

File: client/src/components/User/Details/index.jsx
Comment (warmachine028) [inline client/src/components/User/Details/index.jsx:1]: Why did you delete this file? Was it by accident?

**Review by warmachine028 — APPROVED** [review]

**Review by warmachine028 — CHANGES_REQUESTED** [review]
Screenshot_2023-10-08-21-32-37-742_com android chrome

- Can you add a thin border of greyish color in the dark mode only to these cards, they are very difficult to distinguish in the dark mode.

- Also please merge the latest changes to resolve the merge conflict occuring in the file.

**Comment by waleedkhano** [comment]
Added border to card

Screenshot 2023-10-08 213849

**Comment by warmachine028** [comment]
Also resolve the conflicts too

**Review by warmachine028 — APPROVED** [review]
Changes Look good. 
If you resolve the merge conflicts, I'll merge it to main right away

**Comment by warmachine028** [comment]
No worries I have resolved the merge conflict. Happy hacktoberfest 🎉

**Pull request merged**